api-reference/10 UI Widgets/dxGantt/1 Configuration/validation/autoUpdateParentTasks.md

```diff
@@ -5,7 +5,15 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to recalculate the parent task's duration and progress when its child tasks are modified.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+The **Gantt** supports the following rules for parent-child relationships validation:
```

Comment:
The full description still features the word "validation".

Reply:
updated